docs/deploy_streamlit_app.md

```diff
@@ -1,3 +1,156 @@
-# Deploy a Streamlit app
+# Deploy an app
 
-Streamlit sharing is coming soon! Sign up for the beta at [https://www.streamlit.io/sharing-sign-up](https://www.streamlit.io/sharing-sign-up)
+Now that you've created your app, you're ready to share it! Use **Streamlit sharing** to share it with the world completely for free. Streamlit sharing is the perfect solution if your app is hosted in a public GitHub repo and you'd like anyone in the world to be able to access it. If that doesn't sound like your app, then check out [Streamlit for Teams](http://streamlit.io/for-teams) for more information on how to get secure, private sharing for your apps.
+
+Of course, if you want to host your app using another hosting provider, go for it! Streamlit apps work anywhere a Python app works. You can find guides for other hosting providers on our [community-supported deployment wiki](https://discuss.streamlit.io/t/streamlit-deployment-guide-wiki/5099).
+
+## Get a Streamlit sharing account
+
+To get started, first request an invite at [streamlit.io/sharing](http://streamlit.io/sharing). Streamlit sharing is currently invitation-only while we ramp things up. We promise to get you one very quickly ❤️
+
+Once you have your invite you're ready to deploy! It's really straightforward, just follow the next few steps.
+
+## Put your Streamlit app on GitHub
+
+Make sure it's in a public folder and that you have a [requirements.txt file](https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format)
+
+- If you need to generate a requirements file, try using `pipreqs`
+
+```bash
+pip install pipreqs
+pipreqs /home/project/location
+```
+
+```eval_rst
+.. note:: You should only include packages in requirements.txt that are not distributed with a standard Python installation (i.e. only packages that need to be installed from PyPI/conda). If you `include any of these modules from base Python <https://docs.python.org/3/py-modindex.html>`_ in your requirements.txt file, you will get an error when you try to deploy.
```

Comment:
How about "need to be installed with `pip` or `conda`" ?

Normal people don't know what PyPI is.